package.json

```diff
@@ -32,7 +32,7 @@
     "puppeteer": "^18.0.0"
   },
   "engines": {
-    "node": ">=14.15 <18"
+    "node": ">=14.15 <19"
```

Comment:
I believe there is an issue with running lighthouse v9 on node 18+

The reason we added the `<18` was because of that. I am okay with removing `17.x` in the workflow file but leave `<18` here for that reason

Reply:
it might not be a problem anymore - https://github.com/GoogleChrome/lighthouse/pull/13874

I would run lighthouse with node 18 just to make sure

Reply:
Good call!

I've just tested using 18.10.0 – running and generating reports is working as expected. I tested the standalone lighthouse module, and again using the netlify plugin.